eureka_ml_insights/data_utils/transform.py

```diff
@@ -544,3 +544,39 @@ def _extract_usage(self, row, usage_completion_read_col):
         if not pd.isna(row[self.usage_column]) and usage_completion_read_col in row[self.usage_column]:
             return row[self.usage_column][usage_completion_read_col]
         return np.nan
+
+
+@dataclass
+class CleanCOTAnswer(DFTransformBase):
+    """
+    Transform to strip out anything before and including the </think> tag in the model response
+    """
+
+    model_output_column: str
+    model_answer_column: str
```

Comment:
could you please make the tag an argument since this is a general transform now?